### ISSUE TITLE
Can't access DynamoDB Local Shell

### ISSUE THREAD

**Issue by williamhaley** [localstack/localstack#675]
The localstack [change log](https://github.com/localstack/localstack#change-log) shows that localstack uses [DynamoDB Local](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html) under the hood as of v0.5.0

> ... replace dynalite with DynamoDBLocal ...

DynamoDB Local is super useful, and has a [built in web GUI](https://aws.amazon.com/blogs/aws/sweet-treats-for-dynamodb-users/) accessible at http://localhost:8000 by default. This works great when using DynamoDB Local _standalone_.

```
# Launch DynamoDB Local _without_ localstack
java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb
# navigate to http://localhost:8000/shell
```

Since localstack uses DynamoDB Local under the hood. I was hoping that same web UI shell would be accessible when running Dynamo via localstack. However, I'm having issues with that.

```
docker run -p 4569:4569 localstack/localstack:latest
```

When I try to hit http://localhost:4569/shell in a browser the connection times out.

I see [other users can use the DynamoDB shell in localstack](https://github.com/localstack/localstack/issues/294) so clearly it _can_ work, but when I try running using the latest image, I just get an error every time I hit http://localhost:4569/shell

```
2018-03-21T17:43:31:ERROR:localstack.services.generic_proxy: Error forwarding request: expected string or buffer Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 181, in forward
    path=path, data=data, headers=forward_headers)
  File "/opt/code/localstack/localstack/services/dynamodb/dynamodb_listener.py", line 35, in forward_request
    data = json.loads(to_str(data))
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
TypeError: expected string or buffer
```

I tried [explicitly running kinesis with dynamo](https://github.com/localstack/localstack/issues/294), but that made no difference. I encountered the same issues. Same thing if I tried running DynamoDB on a different port using `SERVICES=dynamodb:8000`

I tried some of the other [suggested ways of starting up localstack](https://github.com/localstack/localstack#installing) to no avail.

```
# Using the localstack binary to spin up the docker images
pip install localstack
TMPDIR=/private$TMPDIR localstack start --docker

# Running the infrastructure from the localstack binary
pip install localstack
localstack start
```

Regardless of how I start localstack I see hints that DynamoDB Local is running on 4569, but I can't hit the shell.

```
Starting mock DynamoDB (http port 4569)...
```

Am I just missing something? Am I using the wrong port?

**Comment by williamhaley** [comment]
I got it. The shell web GUI is exposed at `4564`.

```
docker run -p 4569:4569 -p 4564:4564 localstack/localstack:latest
```

 That port does not seem to be documented anywhere. I only noticed it when running localstack with the `--debug` flag.

```
localstack --debug start
```

 I can understand why it's not documented since the shell probably isn't needed for most use cases, but I was more curious as to why I couldn't find it and if something was wrong with my localstack's DynamoDB Local since it seemed to be broken.
